bolt/org_config_test.go

```diff
@@ -0,0 +1,992 @@
+package bolt_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/chronograf"
+)
+
+func TestOrganizationConfig_FindOrCreate(t *testing.T) {
+	type args struct {
+		organizationID string
+	}
+	type wants struct {
+		organizationConfig *chronograf.OrganizationConfig
+		err                error
+	}
+	tests := []struct {
+		name     string
+		args     args
+		addFirst bool
+		wants    wants
+	}{
+		{
+			name: "Get non-existent default config from default org",
```

Comment:
How do we know that it was saved in bolt after we execute the `FindOrCreate`?

Reply:
Fixed & added test